src/components/DatePickerInline/Month.tsx

```diff
@@ -140,6 +146,7 @@ function Month(props: MonthSingleProps | MonthRangeProps | MonthMultiProps) {
                     generatedDays={generatedDays}
                     disableWeekDays={disableWeekDays}
                     onPressDate={onPressDate}
+                    style={isDocked && weekContainerStyle}
```

Comment:
no`isDocked` either you have weekContainerStyle or not.

Reply:
It is handling the month spacing for docked because we do need some spacing in case of docked picker. @shashank1010

Reply:
changed